app/controllers/api/v0/competitions_controller.rb

```diff
@@ -205,8 +205,8 @@ def update_wcif
 
   private def require_scope!(scope)
     require_user!
-    if current_api_user # If we deal with an OAuth user then check the scopes.
-      raise WcaExceptions::BadApiParameter.new("Missing required scope '#{scope}'") unless doorkeeper_token.scopes.include?(scope)
+    if current_api_user && doorkeeper_token.scopes.exclude?(scope) # If we deal with an OAuth user then check the scopes.
```

Comment:
Not sure if Doorkeeper token has an `exclude` method because it's not an array

Reply:
it does `include Enumerable` though?